src/main/java/business/controllers/LabRequestController.java

```diff
@@ -264,8 +261,20 @@ public LabRequestRepresentation sending(UserAuthenticationToken user,
         }
 
         labRequestService.updateStatus(labRequest, Status.SENDING);
-
         labRequest.setSendDate(new Date());
+
+        JSONObject jObject = new JSONObject(body);
+        String returnText = (String) jObject.get("returnDate");
+
+        DateFormat format = new SimpleDateFormat("yyyy-mm-dd'T'HH:mm:ss.SSS'Z'", Locale.ENGLISH);
+        try {
+            Date returnDate = format.parse(returnText);
+            labRequest.setReturnDate(returnDate);
+        }
+        catch(Exception e) {
+            log.error("Can't parse +" + returnText);
```

Comment:
Shouldn't this then throw an `InvalidRequest` (which will result in 400: Bad request).